.github/ISSUE_TEMPLATE/writing_review.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Writing Review
+about: Request feedback on your writing
+title: '[Writing] '
+labels: 'Status:1-New, Type:Feature'
+assignees: ''
+
+---
+# Review Details
+<!-- Review guide for requesting a review: https://arfc.github.io/manual/guides/katy/revreq. -->
+
+**Submission Deadline**: <!-- Indicate if self-imposed or external. -->
+
+**Intended Venue**:
+
+**Link to pdf**:
+
+**Options for receiving feedback**: <!-- Default to however the reviewer is willing to provide it. -->
+
+**Anticipated response time to feedback**: <!-- How long will it take for you to respond to feedback? -->
+
+**Have you applied the [Writing Checklist](https://arfc.github.io/manual/guides/writing/checklist/)?**:
+
+
+## What is the pre-discussed timeline?
```

Comment:
```suggestion
```

I think this somewhat overlaps with `Submission Deadline`.

Reply:
I can see that, this might be a situation where we can work on the wording.

What I was going for is the `Submission Deadline` is just the very last day it can be submitted, the reason I wanted to make it explicit is that I didn't want it to be confused with someone's individual goals. It's probably a little redundant like you said, but I think it could help with clarity? Maybe not though?

Reply:
Ah, I see what you mean. Maybe we could change `Submission Deadline` to `Submission Deadline(s)`? That way it could contain a preferred review time and then a hard deadline for when it's needed?

Reply:
That could work, or we could get rid of the submission deadline line and make a comment in the timeline section that says something like `Make sure to indicate all deadlines and note whether they are required`?

Reply:
Yeah, I think that would work well!